nautobot/core/api/metadata.py

```diff
@@ -284,6 +286,10 @@ def determine_metadata(self, request, view):
         """This is the metadata that gets returned on an `OPTIONS` request."""
         metadata = super().determine_metadata(request, view)
 
+        # Include the object type label for this model.
+        object_type = view.queryset.model._meta.label_lower if hasattr(view, "queryset") else "unknown"
```

Comment:
The default for some of our generic views is `queryset = None` so this should be changed to:

```suggestion
        object_type = view.queryset.model._meta.label_lower if getattr(view, "queryset", None) else "unknown"
```